extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl

```diff
@@ -92,11 +92,13 @@ $(function() {
       $('#needinfo_from_container').removeClass('bz_default_hidden');
       $('#needinfo_from').prop('disabled', false);
       $('#needinfo_role_identity').text('');
+      $('needinfo_from_textfield').prop('required', true);
```

Comment:
Are these changes meant to be included in this pull request? (They might go away if another one is merged, perhaps?)